pkg/apis/monitoring/v1/types.go

```diff
@@ -1030,7 +1030,7 @@ type RelabelConfig struct {
 	Replacement string `json:"replacement,omitempty"`
 	//Action to perform based on regex matching. Default is 'replace'.
 	//uppercase and lowercase actions require Prometheus >= 2.36.
-	//+kubebuilder:validation:Enum=replace;keep;drop;hashmod;labelmap;labeldrop;labelkeep;uppercase;lowercase
+	//+kubebuilder:validation:Enum=replace;Replace;keep;Keep;drop;Drop;hashmod;HashMod;labelmap;LabelMap;labeldrop;LabelDrop;labelkeep;LabelKeep;lowercase;Lowercase;uppercase;Uppercase
```

Comment:
Not feeling strongly but LowerCase and UpperCase look better to my eyes...

```suggestion
	//+kubebuilder:validation:Enum=replace;Replace;keep;Keep;drop;Drop;hashmod;HashMod;labelmap;LabelMap;labeldrop;LabelDrop;labelkeep;LabelKeep;lowercase;LowerCase;uppercase;UpperCase
```

Reply:
I thought so as well but I kept it similar to https://github.com/prometheus/prometheus/blob/0c3abdc26d4cd2e44549b122799c652c9796f8d8/model/relabel/relabel.go#L43-L59 but can update like this if thats better

Reply:
All good then, it makes more sense to be consistent with prometheus/prometheus.